solr/core/src/test/org/apache/solr/response/TestPrometheusResponseWriter.java

```diff
@@ -34,57 +38,104 @@
 import org.junit.ClassRule;
 import org.junit.Test;
 
-/** Tests the {@link PrometheusResponseWriter} behavior */
-@LuceneTestCase.AwaitsFix(bugUrl = "https://issues.apache.org/jira/browse/SOLR-17368")
 public class TestPrometheusResponseWriter extends SolrTestCaseJ4 {
-
   @ClassRule public static SolrJettyTestRule solrClientTestRule = new SolrJettyTestRule();
   public static JettySolrRunner jetty;
+  public static final List<String> VALID_PROMETHEUS_VALUES = Arrays.asList("NaN", "+Inf", "-Inf");
 
   @BeforeClass
   public static void beforeClass() throws Exception {
     solrClientTestRule.startSolr(LuceneTestCase.createTempDir());
     jetty = solrClientTestRule.getJetty();
     solrClientTestRule.newCollection().withConfigSet(ExternalPaths.DEFAULT_CONFIGSET).create();
     jetty.getCoreContainer().waitForLoadingCoresToFinish(30000);
```

Comment:
can call solrClientTestRule.getCoreContainer() instead and same below; maybe put in a var "cc" (a popular abbreviation in Solr for this super-common class)